pgdb/tools.go

```diff
@@ -86,22 +87,22 @@ func upsertModelInfo(definitionFile string, ac *config.APIConfig, db *sqlx.DB) e
 
 	collectionID, err := getCollectionID(tx, definitionFile)
 	if err != nil {
-		fmt.Println("Collection ID:", collectionID, err)
+		log.Println("Collection ID:", collectionID, err)
 		return err
 	}
 
 	modelID, err := upsertModel(tx, rm, definitionFile, collectionID)
 	if err != nil {
 		fmt.Println("Model ID:", modelID, "Name|", definitionFile)
```

Comment:
If you are switching every logging message from fmt print to log print, you should switch these too so we can track the model info in the log file

Reply:
I have only switched the errors to logs (not the general print statements) so that they are reported as stderr in the docker logs. The purpose of this was to identify errors in the container logs. 

There will be a separate PR soon to log and report errors more elegantly.